data_import/templatetags/data_import.py

```diff
@@ -0,0 +1,20 @@
+from django import template
+
+from ..utils import app_name_to_user_data_model
+
+register = template.Library()
+
+
+@register.simple_tag
+def source_is_connected(source, user):
+    """
+    Given 'go_viral', return True or False.
```

Comment:
This description seems a bit rough :)
(although it does seem clear from the name of the tag)